ffcx/codegeneration/definitions.py

```diff
@@ -73,27 +73,43 @@ def coefficient(self, t, mt, tabledata, quadrature_rule, access):
 
         if ttype == "zeros":
             logging.debug("Not expecting zero coefficients to get this far.")
-            return []
+            return [], []
 
         # For a constant coefficient we reference the dofs directly, so no definition needed
         if ttype == "ones" and end - begin == 1:
-            return []
+            return [], []
 
         assert begin < end
 
         # Get access to element table
         FE = self.symbols.element_table(tabledata, self.entitytype, mt.restriction)
-
         ic = self.symbols.coefficient_dof_sum_index()
-        dof_access = self.symbols.coefficient_dof_access(mt.terminal, ic * bs + begin)
+
         code = []
+        pre_code = []
```

Comment:
Could you add a generated comment saying ~this are temporary arrays with unit stride for coefficient w_foo?

Reply:
Added a comment before the creation of the auxiliary array.